### DISCUSSION TITLE
How to use env file in different environments

### DISCUSSION THREAD

**Discussion by zyyv** [nuxt/nuxt#17599] [Questions]
<img width="270" alt="image" src="https://user-images.githubusercontent.com/42139754/168459499-f57c6a5e-ccc9-410d-9a53-210f088ee7f1.png">
There are my environment variables in different environments, i want to change `process.env.NODE_ENV` to read right `.env.xxx` config file. 

How to configure it to run correctly?  Thanks.

**Comment by TheAlexLichter** [comment]
Let's keep the discussion up in https://github.com/nuxt/framework/discussions/3620 where the same question has been asked as well. Unfortunately, there is no answer there yet either.
